### DISCUSSION TITLE
Invalid copilot token: missing token: 403 in Visual Sturio

### DISCUSSION THREAD

**Discussion by ReyesSanchezCastro** [community/community#106394] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I can't connect to Github copilot in Visual Studio, I got the error: Invalid copilot token: missing token: 403.
I've tried to unnistall and install again the extension but I got the same error. What can I do?

**Comment by AidenJamesYt** [comment]
GitHub servers were having issues yesterday. Try it now

**Comment by Akash1134** [comment, marked as answer]
Hi @ReyesSanchezCastro ,

Please logout/login to your account to authenticate and that should hopefully fix the issue. 

For detailed instructions, see [Configure Copilot in your IDE](https://docs.github.com/en/copilot/configuring-github-copilot/configuring-github-copilot-in-your-environment?tool=visualstudio)

Thanks!

**Comment by ReyesSanchezCastro** [comment]
I've tried to do it and same error. I logout/login in github and visual
studio too, but nothing different happens.

**Reply by ayim** [reply]
Could you record a screenshare or paste the complete logs from the View > Output section?

**Comment by HelloDuoLA** [comment]
> Could you record a screenshare or paste the complete logs from the View > Output section?



> Could you record a screenshare or paste the complete logs from the View > Output section?

I ran into the same problem, this is the output log.

2024-03-18 08:56:59.132 [info] [auth] Invalid copilot token: missing token: 403 
2024-03-18 08:56:59.194 [error] [default] Extension activation failed: "Contact Support. You are currently logged in as HelloDuoLA."

**Reply by alalshow** [reply]
안녕하세요 이 문제 해결하셨나요?
